### PR TITLE
tweakREADME

### PR DESCRIPTION
**Pull request by monopole**

### CODE REVIEW

File: cmd/kinflate/README.md [hunk 1]
```diff
@@ -32,21 +30,27 @@ GOBIN=$tmp go get k8s.io/kubectl/cmd/kinflate
...
+$tmp/kinflate setprefixname acme-
 
-# See the change:
-grep prefixName Kube-manifest.yaml
+# See the added line:
+grep namePrefix Kube-manifest.yaml
+git diff Kube-manifest.yaml
 
 # Send the app to your cluster:
 $GOBIN/kinflate inflate -f . | kubectl apply -f -
```
Comment (Liujingfang1) [inline cmd/kinflate/README.md]: You can change `$GOBIN/kinflate` this to `$tmp/kinflate` to keep consistency with upper lines.

File: cmd/kinflate/README.md [hunk 2]
```diff
@@ -32,21 +30,27 @@ GOBIN=$tmp go get k8s.io/kubectl/cmd/kinflate
...
 # Download a demo app
-git clone https://github.com/monopole/kapps/demo $tmp
+cd $tmp
+git clone https://github.com/monopole/damapps
+cd damapps/demo
 
 # Use kinflate to customize it
-cd $tmp
-$GOBIN/kinflate setprefixname acme-
+$tmp/kinflate setprefixname acme-
```
Comment (Liujingfang1) [inline cmd/kinflate/README.md]: I suggest we add some output for the `kinflate setprefixname` command to indicate what change has been performed.

File: cmd/kinflate/README.md (hunk 1 above)
Comment (monopole) [reply cmd/kinflate/README.md]: done

File: cmd/kinflate/README.md (hunk 2 above)
Comment (monopole) [reply cmd/kinflate/README.md]: that would require doing so for all commands, and assuring that the message remained consistent with the command's help.

**Comment by Liujingfang1** [comment]
/LGTM. Thank you for this change.

**Pull request merged**